app/helpers/application_helper.rb

```diff
@@ -2,4 +2,11 @@ module ApplicationHelper
   def get_article_types
```

Comment:
It would be worth keeping it in the controller if we ever think this will change, then we have a central location to change it from